libs/sdk-backend-tiger/src/backend/uiFeatures.ts

```diff
@@ -61,6 +64,7 @@ export const DefaultFeatureFlags: ITigerFeatureFlags = {
     enablePushpinGeoChart: false,
     enableAttributeAllValues: false,
     enableKPIAttributeFilterRenaming: false,
+    enableKPIDashboardExportPDF: false, // can be set to true after sdk v8.12 codedrop and backends fully support that
```

Comment:
We could probably add a TODO link with the Jira ticket here.

Reply:
done in 7fa283bc50cd8e8f751cbdd8620cc5074ef44a4f